docs/tutorials.rst

```diff
@@ -0,0 +1,38 @@
+Tutorials
+=========
+
+
+# TODO(zach) move the following section somewhere useful
```

Comment:
I agree, maybe a developer guide, or just a general architecture and background section would be good here.  It will probably be more clear after there is at least one tutorial.